Documentation/GeneralConventions/FileStructure.rst

```diff
@@ -8,49 +8,32 @@ File structure
 ==============
 
 This page explains the general file structure of a TYPO3 documentation that can
-be rendered with the :ref:`rendering toolchain <rendering-docs>`. The toolchain
-itself uses a `Sphinx theme <https://typo3-documentation.github.io/sphinx_typo3_theme/>`__
-for converting reStructuredText (reST) or Markdown to HTML.
+be rendered with the :ref:`rendering toolchain <rendering-docs>`.
 
 ..  contents:: Table of Contents:
     :backlinks: top
     :class: compact-list
     :depth: 2
     :local:
 
-
 .. _file-structure-general:
 
 General
 =======
 
 In order for the documentation to be rendered, you need at least
 
-#.  an index file in one of the following locations in weighted order:
-
-    -  :file:`Documentation/Index.rst`
-    -  :file:`Documentation/index.rst`
-    -  :file:`Documentation/Index.md`
-    -  :file:`Documentation/index.md`
-    -  :file:`README.rst`
-    -  :file:`README.md`
-
-#.  and a theme configuration file under
-
-    - :file:`Documentation/Settings.cfg`.
+*   A valid :file:`composer.json`
+*   A documentation entry point either for a :ref:`multi page manual <full-documentation>`
+    or for the rendering of a :ref:`single README file <single-file-documentation>`.
 
 Further conventions are:
 
-*  reST files have the extension **.rst**.
-*  Markdown files have the extension **.md**.
-*  Included reST files have the extension **.rst.txt**.
-*  Use **CamelCase** for directory and file names,
-   for example: Documentation/GeneralConventions/FileStructure.rst.
-
-These conventions pave the way for the *full documentation* and the
-*single file documentation* style, the first being preferred for several
-reasons.
-
+*   reST files have the extension :file:`.rst'.
+*   Markdown files have the extension :file:`.md'.
+*   Included reST files have the extension :file:`.rst.txt'.
+*   Use **CamelCase** for directory and file names,
```

Comment:
I would distinguish between **convention** (CamelCase, but can be handled also in another way, like lowercase dashed).

And a "must-have", like ".rst" and ".md" - this is not a convention but a **requirement**. Otherwise it will not be rendered.

Reply:
done